packages/financial-templates-lib/src/clients/ExpiringMultiPartyClient.js

```diff
@@ -24,7 +24,8 @@ class ExpiringMultiPartyClient {
     empAddress,
     collateralDecimals = 18,
     syntheticDecimals = 18,
-    priceFeedDecimals = 18
+    priceFeedDecimals = 18,
+    contractType
```

Comment:
the client now uses the `contractType` to make small changes to the internal logic.